Gemfile

```diff
@@ -2,3 +2,9 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in action_subscriber.gemspec
 gemspec
+
+if RUBY_ENGINE == "ruby" && RUBY_VERSION.split(".").first.to_i < 2
```

Comment:
@film42 I don't think we can pull this back into the gemspec. The gemspec is evaluated when we are building the gem. So we can't have a version of the gemspec specific for MRI 1.9. But we should probably document that if you need to run MRI 1.9.X series they will have to add these specifications themselves